docs/reference/ml/df-analytics/apis/put-trained-model-definition-part.asciidoc

```diff
@@ -0,0 +1,57 @@
+[role="xpack"]
+[testenv="basic"]
+[[put-trained-model-definition-part]]
+= Put trained model definition part API
```

Comment:
The title and the nav title currently do not match (and don't match the other "create trained models" API), so I recommend changing unless there's a strong reason for sticking with "put":
```suggestion
= Create trained model definition part API
```